examples/log.c

```diff
@@ -347,6 +347,11 @@ static void print_commit(git_commit *commit)
 	git_oid_tostr(buf, sizeof(buf), git_commit_id(commit));
 	printf("commit %s\n", buf);
 
+	if (show_log_size) {
+		printf("log size %d", (int)strlen(git_commit_message(commit)));
+		printf("\n");
```

Comment:
Why not just one printf call?

~~~
printf("log size %d\n", (int)strlen(git_commit_message(commit)));
~~~